mail-client/scooby/scooby-0.9.9.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="Full featured e-mail client"
+DESCRIPTION="Scooby is an open source full featured BeOS native e-mail client."
+HOMEPAGE="https://sourceforge.net/p/scooby/"
+COPYRIGHT="Atsushi Takamatsu"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="e53f65326329dd5a85fbdee640f4edefc2d043d3"
+SOURCE_URI="https://github.com/HaikuArchives/Scooby/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="29630d6ef1ff51cb463762d9bed23c8ebadac045e023c28b1ce1e8d72469384c"
+SOURCE_FILENAME="$portVersionedName.tar.gz"
+SOURCE_DIR="Scooby-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
@nysnatuss, since the package only builds on x86_gcc2, here's what you can do:
Replace
```
ARCHITECTURES="x86_gcc2 x86 x86_64"
```
by
```
ARCHITECTURES="x86_gcc2 !x86 !x86_64"
SECONDARY_ARCHITECTURES="x86_gcc2"
```
and **keep** all the occurences of secondaryArchSuffix.

You should also replace
```
SOURCE_FILENAME="$portVersionedName.tar.gz"
```
by
```
SOURCE_FILENAME="scooby-$portVersion.tar.gz"
```